src/sql/upgrade-42.sql

```diff
@@ -0,0 +1 @@
+alter table Builds add column isChannel integer not null default 0;
```

Comment:
BTW we should probably use `boolean` for these columns, since we don't support SQLite anymore. The only problem is that SQLite is still used to generate the DBIx::Class files, but we can just start a temporary PostgreSQL instance for that.